components/card/README.md

```diff
@@ -82,44 +193,66 @@ The `d2l-card-footer-link` element is an icon link that can be placed in the `fo
 | `target` | String | Where to display the linked URL |
 
 See the [anchor element docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a) for more information on standard link attributes and their values.
+<!-- docs: end hidden content -->
 
-## d2l-card-content-title
+## Card content Title <d2l-card-content-title>
```

Comment:
```suggestion
## Card Content Title <d2l-card-content-title>
```

Reply:
Maybe all these card contents should be something like `Card Content: Title`, `Card Content: Footer` etc